addons/website/views/snippets/snippets.xml

```diff
@@ -540,7 +540,7 @@
     <t t-set="so_content_addition_selector" t-translation="off">blockquote, .s_card:not(.s_timeline_card), .s_alert, .o_facebook_page, .s_share, .s_rating, .s_hr, .s_google_map, .s_map, .s_countdown, .s_chart, .s_text_highlight, .s_progress_bar, .s_badge, .s_embed_code, .s_donation</t>
     <div id="so_content_addition"
         t-att-data-selector="so_content_addition_selector"
-        t-attf-data-drop-near="p, h1, h2, h3, ul, ol, .row > div > img, #{so_content_addition_selector}"
+        t-attf-data-drop-near="p, h1, h2, h3, ul, ol, .row > div > img, section #{so_content_addition_selector.replace(',', ', section')}"
```

Comment:
This is too fragile:
- if someone extends the so_content_addition_selector list with `s_xxx:not(.hello, .world)` (which is a valid jquery selector), the `,` should not be replaced like you did
- "section" is not necessary handling all cases (`so_snippet_addition_selector` targets more than section and could target even more with customizations) and I don't think it matters what there is an ancestor section it is more "is the parent of the neighbor equal to the so_snippet_addition's data-drop-in I think" :grimacing: Maybe it's worth adding a "data-drop-near-exclude-parent" or something like that?